python/ray/_private/worker.py

```diff
@@ -2449,32 +2453,34 @@ def show_in_dashboard(message: str, key: str = "", dtype: str = "text"):
 
 @overload
 def get(
-    object_refs: "Sequence[ObjectRef[Any]]", *, timeout: Optional[float] = None
+    object_refs: "Sequence[RayWaitable[Any]]", *, timeout: Optional[float] = None
 ) -> List[Any]:
     ...
 
 
 @overload
 def get(
-    object_refs: "Sequence[ObjectRef[R]]", *, timeout: Optional[float] = None
+    object_refs: "Sequence[RayWaitable[R]]", *, timeout: Optional[float] = None
 ) -> List[R]:
     ...
 
 
 @overload
-def get(object_refs: "ObjectRef[R]", *, timeout: Optional[float] = None) -> R:
+def get(object_refs: "RayWaitable[R]", *, timeout: Optional[float] = None) -> R:
     ...
 
 
 @PublicAPI
 @client_mode_hook
 def get(
-    object_refs: Union[ray.ObjectRef, Sequence[ray.ObjectRef]],
+    object_refs: Union["RayWaitable[Any]", Sequence["RayWaitable[Any]"]],
```

Comment:
Given passing StreamingObjectRefGenerator is not allowed, we shouldn't change to RayWaitable here right?

Reply:
Same answer as above. If there's a suggestion to make this work, I'd love to hear that

Reply:
Addressed